torch/csrc/jit/passes/quantization.cpp

```diff
@@ -852,6 +864,24 @@ void InsertQuantDeQuantHelper::run(
   script::Method method = module.get_method(method_name);
   auto graph = method.graph();
 
+  // We only need to register new parameters if the graph has
+  // been quantized before
+  if (values_to_quantize_.count(graph.get())) {
+    for (auto& vs : values_to_quantize_.at(graph.get())) {
```

Comment:
Having the same loop here and in `quantizeTensors` is not great. Any ideas how it can be refactored to reduce code duplication?

Reply:
yeah, I thought about this before, the only difference is one place uses `register_attribute` and the other uses `setAttr`, maybe I can have a function with a boolean input to distinguish between these two cases

Reply:
or I can check whether the attribute exists in the module or not and then switch between these two cases